ktor-client/ktor-client-plugins/ktor-client-content-negotiation/common/src/io/ktor/client/plugins/contentnegotiation/ContentNegotiation.kt

```diff
@@ -29,6 +30,11 @@ internal val DefaultCommonIgnoredTypes: Set<KClass<*>> = setOf(
 
 internal expect val DefaultIgnoredTypes: Set<KClass<*>>
 
+/**
+ * Shows that request should skip auth and refresh token procedure.
+ */
+public val ExcludeContentTypes: AttributeKey<List<ContentType>> = AttributeKey("exclude-content-types")
```

Comment:
minor: I would rename it to `ExcludedContentTypes` and it's better to use name `AttributeKey("ExcludedContentTypesAttr")` for consistency